drivers/storage/ide/uniata/CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ list(APPEND SOURCE
     id_queue.cpp
     id_sata.cpp
     ros_glue/ros_glue.cpp
-    stdafx.h)
+    stdafx.h) # Not stdafx.cpp (which only includes the .h), to fix PCH.
```

Comment:
Why comment that you should not add 'stdafx.cpp' here?
Why not also comment that you should not include 'RandomOtherFile.cpp' there?

Reply:
> Why comment that you should not add 'stdafx.cpp' here?

To prevent from the mistake being actually made again.

> Why not also comment that you should not include 'RandomOtherFile.cpp' there?

??

Reply:
> ??

Trying to convey a message with sarcasm doesnt seem to be processed by you, so I'll rephrase it:

There is no need to add random 'Dont do xyz here' when there is no suggestion anywhere to actually do 'xyz'

Reply:
Actually, I can understand @SergeGautherie here, because the other files are *.cpp files as well - which can lead to wrong "fixes".

Reply:
> There is no need to add random 'Dont do xyz here' when there is no suggestion anywhere to actually do 'xyz'

Agreed.
Reverted, though that is not the case here...

Reply:
I don't see a problem.